### ISSUE TITLE
CVE Dio 5.0.3 | Google OVS Scanner

### ISSUE THREAD

**Issue by set0x** [cfug/dio#1752]
### Package

dio

### Version

5.0.3

### Dart Version

2.19.4

### Steps to Reproduce

Excute scanner in a flutter project with dio 5.0.3 dependency.

```bash
osv-scanner .
```

You can use Docker image from project

* https://github.com/google/osv-scanner/pkgs/container/osv-scanner

### Expected Result

Empty response from scanner.

### Actual Result

```
+-------------------------------------+-----------+---------+---------+--------------+
| OSV URL (ID IN BOLD)                | ECOSYSTEM | PACKAGE | VERSION | SOURCE       |
+-------------------------------------+-----------+---------+---------+--------------+
| https://osv.dev/GHSA-jwpw-q68h-r678 | Pub       | dio     | 5.0.3   | pubspec.lock |
+-------------------------------------+-----------+---------+---------+--------------+
```

**Comment by AlexV525** [comment]
#1130 has been resolved in 927f79e already according to dart-lang/sdk@6abb6e5. What should we do next?

**Comment by AlexV525** [comment]
The OSV should no longer report the vulnerability.
